Vejleder kommentarer (#83)
MrLongbottom/P10-Article: sections/related_works.tex

WF-LDA extends \gls{lda} by using word features to make a prior for the topics.
LF-LDA takes the approach of replacing \gls{lda}'s topic-word Dirichlet multinomial component with a two-component mixture of a topic-word Dirichlet multinomial component and a latent feature component.

From these works, we adapt the concepts from \citet{author_topic_2012}\vejleder{argumenter ud fra data + formål} for how we use metadata in our models, though with slight changes in the models for handling the characteristics of our data.
As mentioned earlier, one of the main reasons why we use the concepts from the Author-Topic model instead of the newer MetaLDA model~\cite{MetaLDA2017} is that in MetaLDA, each document has a specific Dirichlet prior for its topic distribution, which is computed from the metadata of the document.\vejleder[inline]{meget vanskeligt at følge med uden indsigt i modellerne}

> Analyzing such a model that includes multiple metadata can be complicated, while in the author-topic model other metadata can be included as their own meta-topic distributions and be analyzed further.

Reviewer: ```suggestion
Analyzing such a model that includes multiple metadata can be complicated, while in the author-topic model, other metadata can be included as their own meta-topic distributions and be analyzed further.
```

WF-LDA extends \gls{lda} by using word features to make a prior for the topics.
LF-LDA takes the approach of replacing \gls{lda}'s topic-word Dirichlet multinomial component with a two-component mixture of a topic-word Dirichlet multinomial component and a latent feature component.

> From these works, we adapt the concepts from \citet{author_topic_2012}\vejleder{argument ad fra data + formål} for how we use metadata in our models, though with slight changes in the models for handling the characteristics of our data.

Reviewer: ```suggestion
From these works, we adapt the concepts from \citet{author_topic_2012}\vejleder{argumenter ud fra data + formål} for how we use metadata in our models, though with slight changes in the models for handling the characteristics of our data.
```

> From these works, we adapt the concepts from \citet{author_topic_2012}\vejleder[inline]{argument ad fra data + formål} for how we use metadata in our models, though with slight changes in the models for handling the characteristics of our data.

Reviewer 2: ```suggestion
From these works, we adapt the concepts from \citet{author_topic_2012}\vejleder{argument ad fra data + formål} for how we use metadata in our models, though with slight changes in the models for handling the characteristics of our data.
```

As mentioned earlier, one of the main reasons why we use the concepts from the Author-Topic model instead of the newer MetaLDA model~\cite{MetaLDA2017} is that in MetaLDA, each document has a specific Dirichlet prior for its topic distribution, which is computed from the metadata of the document.\vejleder[inline]{meget vanskeligt at følge med uden indsigt i modellerne}

> Analyzing such a model that includes multiple metadata, can be complicated while in the author-topic model other metadata can be included as their own meta-topic distributions and be analyzed further.

Reviewer: ```suggestion
Analyzing such a model that includes multiple metadata can be complicated, while in the author-topic model other metadata can be included as their own meta-topic distributions and be analyzed further.
```
